spec.html

```diff
@@ -46873,7 +46869,7 @@ <h1>Promise Resolve Functions</h1>
 
       <emu-clause id="sec-fulfillpromise" type="abstract operation">
```

Comment:
New ID?

Reply:
I don't see a need to always update the IDs to match the text. We're just going to have to oldid it anyway.